content/en/docs/mechanical-assembly/belt-tensioners/index.md

```diff
@@ -0,0 +1,94 @@
+
+6. Repeat this two more times so that you have a total of **three** arms.
+
+## Attaching a tensioner to the left side
+
+Now you'll attach the first tensioner to the {{< tooltip "Front Left Leg" "FDM-0001" >}}:
+
+7. Insert an M5 Nyloc Hex Nut into the recess for a hex nut on the bottom of one of the idler mount arms on the {{< tooltip "Front Left Leg" "FDM-0001" >}}.
+  {{< container-image path="images/7-left-side-nut.png" alt="M5 Nyloc Hed Nut positioned in Front Left Leg" >}}
+
+8. Insert a completed {{< tooltip "Belt Tension Arm "FDM-0037 >}} assembly between the arms on the {{< tooltip "Front Left Leg" "FDM-0001" >}}, with the screw head of the M5x25mm facing up.
+  {{< container-image path="images/8-left-side-arm.png" alt="Belt Tensioner Arm" >}}
+
+9. Insert an M5x40mm Machine Screw through the hole in the top so that it passes through the {{< tooltip "Belt Tension Arm" "FDM-0037" >}} and tightens into the M5 Nyloc Hex Nut. Do not over-tighten, and ensure you can still pivot the tension arm.
+  {{< container-image path="images/9-left-side-bolt.png" alt="M5x40mm screw inserted into Idler and Belt Tensioner Arm" >}}
+  {{< container-image path="images/10-left-side-complete.jpg" alt="Idler and Belt Tension Arm installed on Leg" >}}
+
+## Attaching a tensioner to the right side
+
+Next, attach the second tensioner the the {{< tooltip "Front Right Leg" "FDM-0002" >}}. This is nearly identical to the first one:
+
+10. Insert the M5 Nyloc Hex Nut into the recess for a hex nut on the bottom of one of the idler mount arms on the {{< tooltip "Front Right Leg" "FDM-0002" >}}.
+    {{< container-image path="images/11-right-side-nut.png" alt="M5 Nyloc Hed Nut positioned in Front Left Leg" >}}
+
+11. Insert a completed {{< tooltip "Belt Tension Arm" "FDM-0037" >}} assembly between the arms on the {{< tooltip "Front Right Leg" "FDM-0002" >}}, with the screw head of the M5x25mm facing up.
+  {{< container-image path="images/12-right-side-arm.png" alt="Belt Tensioner Arm" >}}
+
+12. Insert the M5x40mm Machine Screw through the hole in the top so that it passes through the {{< tooltip "Belt Tension Arm" "FDM-0037" >}} and tighten into the M5 Nyloc Hex Nut. Do not over-tighten, and ensure you can still pivot the tension arm.
+  {{< container-image path="images/13-right-side-bolt.png" alt="M5x40mm screw inserted into Idler and Belt Tensioner Arm" >}}
+  {{< container-image path="images/14-right-side-complete.jpg" alt="Idler and Belt Tension Arm installed on Leg" >}}
+
+## Attaching a tensioner to the X idler mount
+
+The third and final tensioner is attached to the {{< tooltip "X Idler Mount" "FDM-0013" >}}. This is only slightly different from the previous two:
+
+13. Insert the M5 Nyloc hex nut into the recess on the bottom of the {{< tooltip "X Idler Mount" "FDM-0013" >}}.
+  {{< container-image path="images/15-x-nut.png" alt="Nyloc hex nut installed on the X Idler Mount" >}}
```

Comment:
FYI I fixed the image reference here, it was pointing to the image used in the next step by accident